tests/aws/services/lambda_/test_lambda.py

```diff
@@ -101,6 +101,10 @@
 )
 TEST_LAMBDA_ENV = os.path.join(THIS_FOLDER, "functions/lambda_environment.py")
 
+TEST_LAMBDA_EVENT_SOURCE_MAPPING_SEND_MESSAGE = os.path.join(
+    THIS_FOLDER, "functions/lambda_event_source_mapping_send_message.py"
+)
+TEST_LAMBDA_SQS_QUEUE_MIRROR = os.path.join(THIS_FOLDER, "functions/lambda_sqs_queue_mirror.py")
```

Comment:
What file is this?

Reply:
Oh damn.  That was a test file I created to mirror an SQS queue from 1 to another using ESM. Turned out to be overkill. Will remove this reference

Reply:
Removed